README.md

```diff
@@ -26,18 +26,39 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
 
 This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
 
-To use this app, you need to import in your dependencies on `manifest.json`.
+This app is used as a block in our [Store Header](https://github.com/vtex-apps/store-header/blob/master/store/interfaces.json). You can configure the minicart properties 
```

Comment:
Looks like here it's missing something. `You can configure the minicart properties`. How? Where?